pkg/bootstrap/docker/openshift/logging.go

```diff
@@ -0,0 +1,140 @@
+package openshift
+
+import (
+	"bytes"
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	genappcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+)
+
+const (
+	loggingNamespace               = "logging"
+	svcKibana                      = "kibana-logging"
+	loggingDeployerAccountTemplate = "logging-deployer-account-template"
+	loggingDeployerTemplate        = "logging-deployer-template"
+)
+
+func instantiateTemplate(client client.Interface, mapper configcmd.Mapper, templateNamespace, templateName, targetNamespace string, params map[string]string) error {
```

Comment:
we already have one of these for the jenkins admission controller, can we not reuse it/move it to a helper utility location?